code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -608,6 +610,7 @@
 	moduleselect_icon = "malf"
 	can_be_pushed = FALSE
 	hat_offset = 3
+	syndicate_module = TRUE
```

Comment:
Proper autodoc please

Reply:
Wrong line, in the declaration of course, my bad

Reply:
What it the format for auto doc comments?

Reply:
Add another forward slash so its /// if you could add it for the rest of the file you would make our lives easier.

Reply:
And after the variable is fine?